### PR TITLE
Wl change rse password

### PR DESCRIPTION
**Pull request by stephamon**
takes over #2097

### CODE REVIEW

**Comment by stephamon** [comment]
@kmsholcomb can you add this to your editing queue?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-37b0dc95de/how-to/).

**Comment by kmsholcomb** [comment]
Got it!

**Comment by kmsholcomb** [comment]
Hi @stephamon. Edits are attached. I edited both articles (this one and the Exchange one) to be as similar as possible. In this one, I have some questions about a warning that might not be a warning, and I refer to the other article's warning for comparison. Anyway, please let me know if you have any questions. Thanks!

[Change a Rackspace Email mailbox password_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1136991/Change.a.Rackspace.Email.mailbox.password_editsKH.docx)
[Change a Rackspace Email mailbox password_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1136990/Change.a.Rackspace.Email.mailbox.password_editsKH.pdf)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c33f48f572/how-to/).

File: content/rackspace-email/change-your-rackspace-email-password.md [hunk 1]
```diff
@@ -1,35 +1,70 @@
...
+audit_date: '2017-02-24'
+title: Change a Rackspace Email mailbox password
 type: article
-created_date: '2017-02-10'
-created_by: Brian Cross
-last_modified_date: '2017-02-21'
-last_modified_by: Brian Cross
+created_date: '2017-06-19'
+created_by: William Loy
+last_modified_date: '2017-07-10]1'
```
Comment (catlook) [inline content/rackspace-email/change-your-rackspace-email-password.md]: this looks funky

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d423fda30a/how-to/)

**Pull request merged**